webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+
+import Loading from 'foremanReact/components/Loading';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import AnsibleVariableOverrides from './components/AnsibleVariableOverrides';
+import './AnsibleHostDetail.scss';
+
+const AnsibleHostDetail = ({ response, status }) => {
+  // https://projects.theforeman.org/issues/32398
+  const [activeTab] = useState('variables');
```

Comment:
If the state doesn't change, why `useState` instead of using the value directly?